package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -20,19 +20,48 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 """
-Reading trajectory with Chemfiles --- :mod:`MDAnalysis.coordinates.chemfiles`
-=============================================================================
+Reading trajectory with *chemfiles* --- :mod:`MDAnalysis.coordinates.chemfiles`
+===============================================================================
 
-Classes to read and write files using the `chemfiles`_ library. This library
-provides C++ implementation of multiple formats readers and writers, the full
-list if available `here <formats>`_.
+MDAnalysis interoperates with the `chemfiles`_ library. The *Chemfiles* C++ library 
+supports an expanding set of file formats, some of which are not natively supported by
+MDAnalysis. Using the *CHEMFILES* reader you can use  `chemfiles`_ for the low-level 
+file reading. Check the list of `chemfile-supported file formats <formats>`_.
 
 .. _chemfiles: https://chemfiles.org
-.. _formats: http://chemfiles.org/chemfiles/latest/formats.html
+.. _formats: https://chemfiles.org/chemfiles/latest/formats.html#list-of-supported-formats
```

Comment:
Looks like for some reason sphinx is throwing a duplicate explicit reference error for `formats`. Probably because the definition here differs from line 146 and 285?

Reply:
Thanks, hopefully fixed in 36671d8.

Reply:
doc tests/rtd passing now